Apps/Common/src/Models/RequestResult.cs

```diff
@@ -27,6 +27,26 @@ namespace HealthGateway.Common.Models
     public class RequestResult<T>
         where T : class
     {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="RequestResult{T}"/> class.
+        /// </summary>
+        public RequestResult()
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="RequestResult{T}"/> class.
+        /// </summary>
+        /// <param name="resourcePayload">The resource payload.</param>
+        /// <param name="resultStatus">The result status.</param>
+        public RequestResult(
```

Comment:
Why is this necessary? I dont see a new RequestResult(params)

Reply:
its not necessary, its a nice to have

Reply:
its being used in the tests!

Reply:
Not sure am following. Is it being used?

Reply:
it just makes initializing a mock RequestResult easier (less lines) not a big deal, but i also don't see why this would be bad